src/core/zowe/core_for_zowe_sdk/profile_manager.py

```diff
@@ -259,10 +260,17 @@ def load(
 
         missing_secure_props = []  # track which secure props were not loaded
 
+        loaded_cfg: dict = {}
+
         for i, (config_type, cfg) in enumerate(config_layers.items()):
+            loaded_cfg = always_merger.merge(loaded_cfg, cfg.profiles)
```

Comment:
I did used the `deepmerge` library so that it can merge each layer as a whole `config`

Reply:
Not all config layers can be merged with the same merger function. See the [Node.js SDK](https://github.com/zowe/imperative/blob/edd7bf378b2007416b1824ea05db93d32a248368/packages/config/src/Config.ts#L552-L569) for how profiles of each layer should be merged:
1. Project and Project User are deep merged into a temporary Project Merged layer
2. Global and Global User are deep merged into a temporary Global Merged layer
3. Add profiles to the Project Merged layer which are only present in Global Merged layer